### PR TITLE
include builder-web in make bldr-run

### PR DESCRIPTION
**Pull request by mwrock**
Signed-off-by: Matt Wrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by nellshamrell** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by christophermaier** [comment]
This is relevant to my current interests 😄 

👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -120,10 +120,7 @@ curl -H "authorization:Bearer 7798a535d5dba1b9fbf9694cb51a1f4ccf3f28d8" -X POST
 
 > Note: some manual setup is required to run a Builder cluster on your host machine. This process will be automated in the future, but for now your mileage may vary.
 
-## Running the habitat depot web site
-
-To run the depot web site, you need to start the builder services with `make bldr-run` as described above. Additionally start the `builder-web` node server. You can find details about setting up a `builder-web` environment and configuring oauth credentials on the `builder-web` [readme](https://github.com/habitat-sh/habitat/blob/master/components/builder-web/README.md).
-
+Also be aware that if you need to run the web services from anyehere other than localhost or port 3000, seethe `builder-web` [readme](https://github.com/habitat-sh/habitat/blob/master/components/builder-web/README.md) for how to configure oath with an alternate endpoint.
```
Comment (raskchanky) [inline CONTRIBUTING.md]: `anyehere` should be `anywhere`

Comment (raskchanky) [inline CONTRIBUTING.md]: Also a space between `see` and `the`

Comment (raskchanky) [inline CONTRIBUTING.md]: `oath` should be `oauth`

Comment (mwrock) [inline CONTRIBUTING.md]: oh man

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/222878744).

**Comment by raskchanky** [comment]
It's also worth noting in this documentation that if you're going to be running the depot inside your VM, you'll need to forward port 3000 to your local host, otherwise browsing to `http://localhost:3000` isn't going to show anything but a connection failure.

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/222878744).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**